src/components/map/MapService.js

```diff
@@ -1667,14 +1671,19 @@ goog.require('ga_urlutils_service');
           if (!extent || extent.length !== 4) {
             return gaGlobalOptions.defaultExtent;
           }
-          return [
+          var extent = [
             Math.max(extent[0], gaGlobalOptions.defaultExtent[0]),
             Math.max(extent[1], gaGlobalOptions.defaultExtent[1]),
             Math.min(extent[2], gaGlobalOptions.defaultExtent[2]),
             Math.min(extent[3], gaGlobalOptions.defaultExtent[3])
           ];
+          if (!isExtentEmpty(extent)) {
+            return extent;
+          }
```

Comment:
From the first discussion with @oterral in #2733, it seems like the most logical thing to do. It works since ol won't try to fit with an `undefined` extent.

If it returns the default extent it might be confusing since the user may not understand why the extent changed. Furthermore, it is not the correct extent for the KML. So I think that `undefined` is a good compromise.

What would be really great is to inform the user that the imported KML is outside the geoportal (somewhere below `Parsing OK`). But that is outside the scope of this PR.

PR update to explicitly return undefined.